connect/api_test.go

```diff
@@ -86,3 +89,44 @@ func TestUpToDateOkay(t *testing.T) {
 		t.Error("Expecting UpToDate()==true, got false")
 	}
 }
+
+func TestGetProduct(t *testing.T) {
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+		w.Write([]byte(testProduct))
```

Comment:
Another thing about using go embed is that now you have to scroll to the top to see which file testProduct comes from.

Reply:
Will be solved by #10